.github/workflows/published_release.yml

```diff
@@ -172,7 +172,50 @@ jobs:
       artifactName: homebrew-formula
       artifactPath: .
 
+  prepare-documentation-update:
+    runs-on: ubuntu-latest
+    needs: [process-release-information,update-releases-yaml]
+    if: endsWith(github.event.client_payload.tagName, '.0')
```

Comment:
Will this also include prereleases? Do we want to include them?

Reply:
what do you mean by prereleases?